### ISSUE TITLE
Hello World TypeScript errors on event-target-shim

### ISSUE THREAD

**Issue by dfabulich** [bikeshaving/crank#93]
```sh
#!/bin/bash -ex

rm -rf tscrank
mkdir tscrank
cd tscrank
npm init -y
npm install @bikeshaving/crank typescript
cat <<EOF > hello.tsx
/** @jsx createElement */
import {createElement} from "@bikeshaving/crank";
import {renderer} from "@bikeshaving/crank/dom";

renderer.render(<div id="hello">Hello world</div>, document.body);
EOF
npx tsc --jsx react --lib dom,es2020 --module es2020 --moduleResolution node hello.tsx
```

```
node_modules/@bikeshaving/crank/esm/events.d.ts:1:48 - error TS2307: Cannot find module 'event-target-shim'.

1 import { EventTarget as EventTargetShim } from "event-target-shim";
                                                 ~~~~~~~~~~~~~~~~~~~


Found 1 error.
```

I believe the problem is that `event-target-shim` is declared in `devDependencies`, but Crank dependents need it in order to satisfy TypeScript. If you `npm install event-target-shim` in the example project, `tsc` runs without errors.

**Comment by brainkim** [comment]
Thanks for catching all these errors. I know it must be incredibly frustrating, and I sincerely apologize for my lack of diligence with regard to to different environments. I’m honestly very new to the task of maintaining an actual frequently used library on npm. My assumption was that by vendoring the dependency with rollup we would get the types out of box? Would adding @types/event-target-shim yourself solve this problem locally? I do like the idea of not adding event-target-shim as a direct dependency because we do plan on pulling it in later.

**Comment by dfabulich** [comment]
No worries! I have a library that I was thinking of releasing soon, and I'm letting Crank catch all of the weird issues that I was going to have to deal with. ;-)

> My assumption was that by vendoring the dependency with rollup we would get the types out of box?

I'm a relative newbie to TypeScript, but I don't think rollup will _ever_ auto-fix a line like this in a `d.ts` file, even if rollup does have its own `EventTarget` types baked in.

```js
import { EventTarget as EventTargetShim } from "event-target-shim";
```

> Would adding @types/event-target-shim yourself solve this problem locally?

I don't think `@types/event-target-shim` is an actual project…?

**Comment by brainkim** [comment]
> I don't think @types/event-target-shim is an actual project…?

You’re right I thought for a second event-target-shim was typed via DefinitelyTyped 😓 I think the solution might be to just pull it in.

**Comment by ryhinchey** [comment]
We should make event target shim a dependency.  We can bring it in eventually but currently crank is dependent on it.  It ships with types

**Comment by brainkim** [comment]
I mean it’s dependent on it but it’s vendored via rollup. I thought that was the whole point of rollup that dependencies could be vendored? I guess it doesn’t make a big difference and we can add it to dependencies instead.

**Comment by ryhinchey** [comment]
the source code is vendored but the types aren't

**Comment by brainkim** [comment]
Shipped 0.1.3
